src/QirRuntime/samples/StandaloneInputReference/qir-driver.cpp

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "CLI11.hpp"
+#include "CoreTypes.hpp"
+#include "QirContext.hpp"
+#include "QirTypes.hpp"
+#include "QuantumApi_I.hpp"
+#include "SimFactory.hpp"
+#include "quantum__qis_internal.hpp"
+#include "quantum__rt.hpp"
+#include <cctype>
+#include <iostream>
+#include <map>
+#include <vector>
```

Comment:
I know that clang-format by default recommends the order of the headers inclusion _opposite_ to the order of headers currently in our files. I know the reasons behind both options.
As I remember, this topic is not adjusted explicitly in our [.clang-format](https://github.com/microsoft/qsharp-runtime/blob/main/src/QirRuntime/.clang-format), at least the option [`SortIncludes`](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) is not specified explicitly.

For the files added by me I preferred to stay with the same order that's in other files.

For your case you may consider 
* either the same or 
* listing the most specific headers first, and then the more general ones (rather than sorting "CLI11.hpp" in the same group as other ".." files). That is you might consider placing "CLI11.hpp" after the other ".." files but before the <.. > files.

Reply:
I changed the order to be consistent with the order of headers currently in our other cpp files.